.gitignore

```diff
@@ -23,3 +22,7 @@ todo.md
 .buildpath
 .project
 .settings
+
+# sublime project files
+*.sublime*
+
```

Comment:
These are no files related to the Tin Can library, but they are files specific to your own setup. Therefore, they should not be part of the project's `.gitignore` file. You can add them to your own global Git ignore settings.